source/ways-of-working/common-pipeline/common-pipeline.html.md.erb

```diff
@@ -47,6 +47,8 @@ For more information about Helm, please see the related new component
 
 ![The Common Pipeline](/images/common-pipeline.png)
 
+![SDS AKS Release](/images/sds-aks-release.png)
```

Comment:
this must be generic and can't have SDS in it's name

Reply:
Updated